tokenizers/examples/unstable_wasm/.appveyor.yml

```diff
@@ -0,0 +1,11 @@
+install:
+  - appveyor-retry appveyor DownloadFile https://win.rustup.rs/ -FileName rustup-init.exe
```

Comment:
Why use appveyor for the CI ?

Reply:
It's in the default template for `wasm` app. I don't really see a lot of downsides of keeping them.

Reply:
I'd leave it up to the user to know or not to add these kinds of files, they don't showcase anything particular to `tokenizers` and are readily available on `wasm` tutorials. But that's just my 2 cents !